README.md

```diff
@@ -35,13 +35,15 @@ My background music for joy, work, and other typing sessions:
 Liminaut, [*a playlist for exploring liminal spaces*](https://www.youtube.com/watch?v=DqjWFZN82_A) by *nobody*
    - Slow ambient [Trackside Isolation (Dark Ambient Mix)](https://www.youtube.com/watch?v=RwF_0HSr_MI) by *Thunderbolt 1000 Siren Productions*
 - 🌌 Highly experimental stuff:
-   - Strange music: [Cryptovolans - DATA - DREAM - DESIRE【SC-88 Pro】](https://www.youtube.com/watch?v=YVNwa_hae_U), 
+   - Strange music: [Cryptovolans - DATA - DREAM - DESIRE【SC-88 Pro】](https://www.youtube.com/watch?v=YVNwa_hae_U),
```

Comment:
Revert an accidental unrelated edit (I tried to place the new entry into Strange Music first):

```suggestion
   - Strange music: [Cryptovolans - DATA - DREAM - DESIRE【SC-88 Pro】](https://www.youtube.com/watch?v=YVNwa_hae_U), 
```